src/components/Task.js

```diff
@@ -1,28 +1,55 @@
 import { Component, PropTypes } from 'react';
+import { createButton } from './../../utils/createTaskButton';
 
 class Task extends Component {
   render() {
-    const { startTask, projectId, id, name, state } = this.props
-    let actionButton = ''
+    const { startTask, completeTask, projectId, id, name, description, state } = this.props;
+    let actionButton = '';
     if (state === 'to-do') {
-      actionButton = <button onClick={e => {
-        e.preventDefault()
-        startTask(projectId, id)
-      }}>Start</button>
+      actionButton = createButton(projectId, id, startTask, 'Start Task', buttonStyle);
+    } else if (state === 'in-progress') {
+      actionButton = createButton(projectId, id, completeTask, 'Complete Task', buttonStyle);
     }
+
     return (
-      <div>
-        <h6 className={state}>{name} {actionButton}</h6>
+      <div style={divStyle}>
+        <p className={state} style={pStyle}>{state === "in-progress" ? `In Progress ${name}` : `Todo ${name}`} {actionButton}</p>
```

Comment:
I agree.  I referenced the original code, and didn't want to step on any toes in the case of someone that might be using the class name.  I'll take it out and update it.